Assets/HoloToolkit/Input/Scripts/Voice/SpeechInputSource.cs

```diff
@@ -0,0 +1,181 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using UnityEngine;
+using UnityEngine.Windows.Speech;
+
+namespace HoloToolkit.Unity.InputModule
+{
+    /// <summary>
+    /// SpeechInputSource allows you to specify keywords and methods in the Unity
+    /// Inspector, instead of registering them explicitly in code.
+    /// This also includes a setting to either automatically start the
+    /// keyword recognizer or allow your code to start it.
+    ///
+    /// IMPORTANT: Please make sure to add the microphone capability in your app, in Unity under
+    /// Edit -> Project Settings -> Player -> Settings for Windows Store -> Publishing Settings -> Capabilities
+    /// or in your Visual Studio Package.appxmanifest capabilities.
+    /// </summary>
+    public partial class SpeechInputSource : BaseInputSource
```

Comment:
I think the only thing blocking this PR is the ability to assign the GameObject and script method to trigger with each of the keywords without hard coding it in a class.
![image](https://cloud.githubusercontent.com/assets/13334553/21238531/725bb4ce-c2d1-11e6-9427-5ef6d440515d.png)

I think the only way to really do this is through a UnityEvent.